ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -0,0 +1,627 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# .. _postldap__ref_defaults:
+
+# debops.postldap default variables [[[
+# =================================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../../includes/global.rst
+
+# APT packages [[[
+# ----------------
+
+# .. envvar:: postldap__postfix__dependent_packages [[[
+#
+# List of additional APT packages needed for the LDAP support.
+postldap__postfix__dependent_packages: '{{ [ "postfix-ldap" ]
+                                                       if postldap__ldap_enabled else [] }}'
+                                                                   # ]]]
+                                                                   # ]]]
+# PKI / TLS configuration [[[
+# ---------------------------
+
+# .. envvar:: postldap__tls_ca_cert_dir [[[
+#
+# Directory containing X509 Certification  Authority  certificates
+# in  PEM  format  which  are  to  be  recognized by the client in
+# SSL/TLS connections. The files each contain one CA  certificate.
+postldap__tls_ca_cert_dir: '/etc/ssl/certs/'
+                                                                   # ]]]
+                                                                   # ]]]
+# Secrets [[[
+# -----------
+
+# .. envvar:: postldap__no_log [[[
+#
+# This variable can be used in the Ansible tasks to set the ``no_log``
+# parameter. Changing it to ``False`` can help with debugging issues with the
+# roles, but otherwise it should be set to ``True`` to ensure that Ansible
+# doesn't leak sensitive data in its logs.
+postldap__no_log: True
+                                                                   # ]]]
+                                                                   # ]]]
+
+# Virtual Mail [[[
+# ----------------
+#
+# This settings help to configure to enable `postfix` to host multiple (virtual) domains,
+# and thus provide email to several domains with just one `mail server`.
+# Currently the Virtual Mail support works only with LDAP enabled, in the future `mariaDB` could be enabled.
+#
+
+# .. envvar:: postldap__vmail_posix_user [[[
+#
+# Virtual Mail POSIX username
+# For the accesses to the mailbox directories a separate user `vmail` (`Virtual Mail`) is created,
+# under which the accesses of Postfix, Dovecot and other components of the mail server should take place.
+# On the one hand this prevents mail server components from accessing sensitive system directories,
+# on the other hand it protects the mailboxes from external access.
+# Only vmail (and root) are allowed to access the mailboxes.
+postldap__vmail_posix_user: 'vmail'
+                                                                   # ]]]
+# .. envvar:: postldap__vmail_posix_uidnumber [[[
```

Comment:
You should add one empty line after the variable and before the `# ]]]` mark to make the variable a bit more readable.

On the next set of variables, you should remove the `#` before the variable for better code readability.

Reply:
> On the next set of variables, you should remove the `#` before the variable for better code readability.

I do not quite understand your comment here. Can you give me a hint, a line number or an example?

Reply:
Sure. For example, you have two variables:

```yaml
# .. envvar:: postldap__vmail_posix_uidnumber [[[
#
# Virtual Mail POSIX uidNumber
#
postldap__vmail_posix_uidnumber: '{{ getent_passwd[postldap__vmail_posix_user][1]|int }}'
                                                                   # ]]]
# .. envvar:: postldap__vmail_posix_group [[[
#
# Virtual Mail POSIX group
#
postldap__vmail_posix_group: 'vmail'
                                                                   # ]]]
```

They should instead look like this:

```yaml
# .. envvar:: postldap__vmail_posix_uidnumber [[[
#
# Virtual Mail POSIX uidNumber.
postldap__vmail_posix_uidnumber: '{{ getent_passwd[postldap__vmail_posix_user][1]|int }}'

                                                                   # ]]]
# .. envvar:: postldap__vmail_posix_group [[[
#
# Virtual Mail POSIX group.
postldap__vmail_posix_group: 'vmail'

                                                                   # ]]]
```

This is probaly more readable?

Reply:
Yeah, thank you! WILCO